src/peakWatcher.h

```diff
@@ -16,3 +20,4 @@ void cmdReportPeakWatcherW2C2(Command* command);
 void cmdResetPeakWatcherW1(Command* command);
 void cmdResetPeakWatcherW2(Command* command);
 void cmdPausePeakWatcher(Command* command);
+void peakWatcherOnSwitchProjectTab();
```

Comment:
Please move this to onSwitchTab in the peakWatcher namespace (alongside initialize). The cmd functions aren't in the namespace, but that's because no cmd function has a namespace.